### DISCUSSION TITLE
Change wallpaper

### DISCUSSION THREAD

**Discussion by nev-21** [cortezaproject/corteza-server#373] [Q&A] [answered]
Hi, Corteza is great. I want to know if it is possible to change the background and the name in the profile settings:

![image](https://user-images.githubusercontent.com/89469783/174912337-8e5bd4d3-1487-48c7-9e56-70a7c9afae28.png)

Thanks in advance

**Comment by zmija** [comment, marked as answer]
Ji @nev-21, thanks for reaching out. The Corteza logo can be replaced in the Admin application (the last item in the sidebar User Interface > Settings).

The background unfortunately can't be replaced from the user interface. If you're running Corteza locally, you can replace this image in the corteza-server repo (/assets/src). Alternatively, check out these instructions: https://docs.cortezaproject.org/corteza-docs/2022.3/integrator-guide/authentication/personalization.html#_export_current_assets

Cheers, Mia

**Reply by nev-21** [reply]
Thanks @zmija
